.github/workflows/continuous-integration-workflow.yml

```diff
@@ -116,4 +123,4 @@ jobs:
         run: |
           cmake -B builddir -DCMAKE_CXX_COMPILER=${{ matrix.cxx }}
           cmake --build builddir
-          cmake --build builddir --target test
+          cmake --build builddir --target test --verbose
```

Comment:
Did you mean to commit this?

Reply:
It is not needed, but actually a verbose build would be better to see all the flags etc.

I am happy to drop it though.